kroxylicious/src/main/java/io/kroxylicious/proxy/Kroxylicious.java

```diff
@@ -43,6 +48,10 @@ public Integer call() throws Exception {
             kafkaProxy.startup();
             kafkaProxy.block();
         }
+        catch (Exception e) {
+            LOGGER.error("Exception on startup", e);
+            throw e;
```

Comment:
maybe better to log at fatal and return a `-XXX` error code?

Reply:
We are using SLF4J here, so there is no `fatal` level.

Java will give us a non-zero exit status for any program that bubbles a exception to main.   

```
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
java.lang.IllegalStateException: Configuration for virtual cluster 'demo2' conflicts with configuration for virtual cluster: 'demo'.
	at io.kroxylicious.proxy.internal.PortConflictDetector.buildOverviewException(PortConflictDetector.java:161)
	at io.kroxylicious.proxy.internal.PortConflictDetector.buildPortConflictException(PortConflictDetector.java:150)
	at io.kroxylicious.proxy.internal.PortConflictDetector.lambda$validate$1(PortConflictDetector.java:64)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1858)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at io.kroxylicious.proxy.internal.PortConflictDetector.lambda$validate$12(PortConflictDetector.java:59)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at io.kroxylicious.proxy.internal.PortConflictDetector.validate(PortConflictDetector.java:45)
	at io.kroxylicious.proxy.KafkaProxy.startup(KafkaProxy.java:94)
	at io.kroxylicious.proxy.Kroxylicious.call(Kroxylicious.java:48)
	at io.kroxylicious.proxy.Kroxylicious.call(Kroxylicious.java:27)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at io.kroxylicious.proxy.Kroxylicious.main(Kroxylicious.java:60)
Caused by: java.lang.IllegalStateException: The exclusive bind of port(s) 9192 to <any> would conflict with existing exclusive port bindings on <any>.
	at io.kroxylicious.proxy.internal.PortConflictDetector.buildPortConflictException(PortConflictDetector.java:144)
	... 32 more

Process finished with exit code 1
```